extensions/metrics/README.md

```diff
@@ -1,7 +1,14 @@
 # @loopback/extension-metrics
 
-This module contains a component to report metrics for
-[Prometheus](https://prometheus.io/).
+This module contains a component that reports metrics of Node.js, LoopBack
+framework, and your application to [Prometheus](https://prometheus.io/).
+
+## Stability - Experimental
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs may subject to breaking changes in future
+> releases.
```

Comment:
> Their APIs may subject to breaking changes in future releases.

I am not sure if this is correct in English, it think the following is better:

```
Their APIs may be subject to (...)
```

In addition to breaking API changes, I'd like to give us the option to introduce breaking changes in behavior too.

Proposed text:

```
Their APIs and functionality may be subject to breaking changes in future releases.
```